src/supplemental/nanolib/conf_ver2.c

```diff
@@ -1028,10 +1028,45 @@ conf_bridge_parse_ver2(conf *config, cJSON *jso)
 	return;
 }
 
+#if defined(SUPP_PLUGIN)
+static void
+conf_plugin_parse_ver2(conf *config, cJSON *jso)
+{
+	cJSON *plugin = hocon_get_obj("plugin", jso);
+	cJSON *libs	      = NULL;
+	cJSON *lib	      = NULL;
+
+	config->plugin.path     = NULL;
+	config->plugin.path_sz  = 0;
+
+	libs = hocon_get_obj("libs", plugin);
+
+	cJSON_ArrayForEach(lib, libs) {
+		cJSON *jso_path = cJSON_GetObjectItem(lib, "path");
+		if (jso_path) {
+			if (cJSON_IsString(jso_path)) {
+				if (jso_path->valuestring != NULL) {
+					log_error("jso_path->valuestring: %s", jso_path->valuestring);
```

Comment:
Donot forget to remove the log when PR is already. Maybe changing its log level to info is also fine

Reply:
TKS, removed